### DISCUSSION TITLE
依然显示白框，以勾选使用国内服务器

### DISCUSSION THREAD

**Discussion by thedespair** [cangzhang/ChampR#213] [General]
怎么都显示不出来
![QQ截图20220915211428](https://user-images.githubusercontent.com/82921802/190415174-4d95bce1-9d26-4be0-958c-02a99540d639.png)

**Comment by thedespair** [comment]
我使用训练模式测试的，之前也出过这个问题，所以一直用1.3.2

**Comment by cangzhang** [comment]
试试能不能在浏览器打开下面的网址：

https://ddragon.leagueoflegends.com/cdn/12.17.1/data/en_US/champion.json

**Reply by thedespair** [reply]
可以打开，有点慢

**Reply by Reckeso** [reply]
选用英雄时无法应用champr的符文是如何解决（应用按钮没用）？谢谢

**Reply by cangzhang** [reply]
确保你的当前符文页是可以被删除的(即当前符文页非系统预置符文), 或者切换到任意一个你的自定义符文, 然后点击应用.

实在不行就删掉所有的自定义符文页, 然后点击应用

**Reply by Reckeso** [reply]
尝试过但没有用，并且我将champr更新了，依旧无法应用符文。

**Reply by cangzhang** [reply]
重新转换一次试试

https://github.com/cangzhang/champ-r/wiki/FAQ#tencent-server-v1218-%E6%97%A0%E6%B3%95%E6%98%BE%E7%A4%BA%E5%A4%A9%E8%B5%8B%E6%8E%A8%E8%8D%90

**Reply by thedespair** [reply]
> 重新转换一次试试
> 
> https://github.com/cangzhang/champ-r/wiki/FAQ#tencent-server-v1218-%E6%97%A0%E6%B3%95%E6%98%BE%E7%A4%BA%E5%A4%A9%E8%B5%8B%E6%8E%A8%E8%8D%90

现在已经可以了，还是需要转换到体验服才可以  
这是转换器的下载连接：
https://www.jb51.net/game/699347.html

**Reply by mzhou0621** [reply]
> 选用英雄时无法应用champr的符文是如何解决（应用按钮没用）？谢谢

我也是一样，以前是偶然失灵，现在完全用不了了，按应用没用
